Tools/build/compute-changes.py

```diff
@@ -57,7 +57,7 @@ class Outputs:
 
 def compute_changes() -> None:
     target_branch, head_ref = git_refs()
-    if target_branch and head_ref:
+    if os.environ.get("GITHUB_EVENT_NAME", "") == "pull_request":
```

Comment:
I think that this should be `and`

```suggestion
    if (
        target_branch 
        and head_ref
        and os.environ.get("GITHUB_EVENT_NAME", "") == "pull_request"
    ):
```

Reply:
See https://github.com/python/cpython/issues/133410#issuecomment-2863238493, the original problem is that `target_branch` and `head_ref` are always be truthy, so I don't think we need to check them.